src/stylesheets/blocks/table/kss/_table-block-kss.html

```
@@ -0,0 +1,139 @@
+<!-- EDS Modal-->
+<div class="env-a-w-100% env-a-w-max-100% env-a-h-550">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)